src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -100,7 +100,16 @@ mod private { // this item is private and will not be documented
 }
 ```
 
-`--document-private-items` documents all items, even if they're not public.
+`--document-private-items` causes all items to be documented as if they were `pub`.
```

Comment:
It's actually not the case. It shows them up with a little lock alongside them, but it's not the same as if they were `pub`.

Reply:
I mean that the logic for whether or not they are included will be the same as that for `pub` items.

The previous wording implied that it also shows `doc(hidden)` items, which it does not.

I am not sure how to word it more clearly without being overly verbose.

Reply:
What do you think about:

> `--document-private-items` includes all non-public in the generated documentation, but not `#[doc(hidden)]` items.

?

Reply:
I would say "except for", but other than that, it should be fine as long as we don't add another separate condition that prevents items from being documented.

Reply:
Sounds good. :)